nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -170,6 +170,9 @@ let
             ssl_certificate ${vhost.sslCertificate};
             ssl_certificate_key ${vhost.sslCertificateKey};
           ''}
+          ${optionalString vhost.forceSSL ''
+            add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
```

Comment:
This should be definitely mentioned in both documention for `forceSSL` and nixos release changelog. There might be also the use case for people wanting a redirect, but no `HSTS` (yet), when testing.